client/asset/btc/btc.go

```diff
@@ -1344,7 +1372,21 @@ func (btc *baseWallet) legacyBalance() (*asset.Balance, error) {
 func (btc *baseWallet) feeRate(_ RawRequester, confTarget uint64) (uint64, error) {
 	feeResult, err := btc.node.estimateSmartFee(int64(confTarget), &btcjson.EstimateModeConservative)
 	if err != nil {
-		return 0, err
+		btc.log.Errorf("Failed to get fee rate with estimate smart fee rate: %v", err)
+
+		if !btc.apiFeeFallback() {
+			return 0, err
+		}
+		btc.log.Debug("Retrieving fee rate from external API: ", externalApiUrl)
+		estimatedFee, err := externalFeeEstimator(btc.ctx, btc.Network)
```

Comment:
Well.... considering https://github.com/decred/dcrdex/issues/1618#issuecomment-1248122923 (@buck54321) we may want to drop this commit from the backport batch.